### PR TITLE
[AU4] Add new empty mono/stereo track

### PR DESCRIPTION
**Pull request by grliszas14**
Resolves: https://github.com/audacity/audacity/issues/7158

### CODE REVIEW

File: au4/src/projectscene/view/clipsview/trackslistclipsmodel.cpp [hunk 1]
```diff
@@ -28,13 +30,27 @@ void TracksListClipsModel::load()
...
     m_trackList = prj->trackList();
 
-    //! TODO Subscribe on tracks changed
-
     m_dataSelectedTracks = selectionController()->dataSelectedOnTracks();
     selectionController()->dataSelectedOnTracksChanged().onReceive(this, [this](const std::vector<trackedit::TrackId>& tracks) {
         setDataSelectedTracks(tracks);
     });
 
+    m_trackList.onItemAdded(m_tracksNotifyReceiver.get(), [this](const trackedit::Track& track) {
```
Comment (igorkorsukov) [inline au4/src/projectscene/view/clipsview/trackslistclipsmodel.cpp:38]: Why did you decide to use a separate object instead of just passing this?

Comment (grliszas14) [reply au4/src/projectscene/view/clipsview/trackslistclipsmodel.cpp:38]: oh, I did it the same way as it is done in TracksListModel -> should I fix that in both?

Comment (igorkorsukov) [reply au4/src/projectscene/view/clipsview/trackslistclipsmodel.cpp:38]: I asked Elnur... in general, he took this solution from MuseScore, it made sense there, but here and in TracksListModel it doesn't make sense :) You can just pass this :)

Comment (grliszas14) [reply au4/src/projectscene/view/clipsview/trackslistclipsmodel.cpp]: fixed

**Pull request merged**